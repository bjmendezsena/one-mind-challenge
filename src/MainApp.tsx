import React, { FC } from "react";

import { Button } from "antd";

import "./MainApp.css";

const MainApp: FC = () => {
  const onClick = () => {
    console.log("Hola mundo");
  };
  return (
    <div>
      <Button type="primary" onClick={onClick}>
        Button
      </Button>
    </div>
  );
};

export default MainApp;
