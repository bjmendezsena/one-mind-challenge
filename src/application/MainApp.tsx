import React, { FC } from "react";
import "./MainApp.css";
import { AppProvider } from "./context/UserContext";
import { MainPage } from "./views/MainPage";

interface IGlobalStateProps {
  children: JSX.Element;
}
const GlobalState = ({ children }: IGlobalStateProps) => (
  <AppProvider>{children}</AppProvider>
);

const MainApp: FC = () => {
  return (
    <GlobalState>
      <MainPage />
    </GlobalState>
  );
};

export default MainApp;
