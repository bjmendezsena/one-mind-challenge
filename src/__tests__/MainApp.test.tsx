import { shallow } from "enzyme";
import MainApp from "../application/MainApp";

describe("MainApp test", () => {
  it("Should render the component", () => {
    const wrapper = shallow(<MainApp />);

    expect(wrapper).toMatchSnapshot();
  });
});
