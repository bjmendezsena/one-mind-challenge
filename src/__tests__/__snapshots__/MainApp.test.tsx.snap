// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainApp test Should render the component 1`] = `
<GlobalState>
  <MainPage />
</GlobalState>
`;
